cluster/gce/configure-vm.sh

```diff
@@ -1129,17 +1034,9 @@ if [[ -z "${is_push}" ]]; then
   set-kube-env
   auto-upgrade
   ensure-local-disks
-  [[ "${KUBERNETES_MASTER}" == "true" ]] && mount-master-pd
```

Comment:
you can also rip out the entire `mount-master-pd` function. Also `find-master-pd`.